.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+build:
+  stage: build
+  image: ghcr.io/acts-project/ubuntu2004_exatrkx:v17
```

Comment:
I think I'd probably also add

```yml
  tags:
    - docker
```

otherwise I believe it isn't *guaranteed* to run on a docker runner.